src/code/z_camera.c

```diff
@@ -574,71 +574,86 @@ s16 Camera_XZAngle(Vec3f* to, Vec3f* from) {
     return CAM_DEG_TO_BINANG(RAD_TO_DEG(Math_FAtan2F(from->x - to->x, from->z - to->z)));
 }
 
-s16 func_80044ADC(Camera* camera, s16 yaw, s16 arg2) {
-    static f32 D_8015CE50;
-    static f32 D_8015CE54;
-    static CamColChk D_8015CE58;
+s16 Camera_GetPitchAdjFromFloorHeightDiffs_(Camera* camera, s16 viewYaw, s16 initAndReturnZero) {
+    static f32 sFloorYSmallOffsetForwards;
+    static f32 sFloorYBigOffsetForwards;
+    static CamColChk sCamColChk_;
```

Comment:
```suggestion
s16 Camera_GetPitchAdjFromFloorHeightDiffs(Camera* camera, s16 viewYaw, s16 initAndReturnZero) {
    static f32 sFloorYSmallOffsetForwards;
    static f32 sFloorYBigOffsetForwards;
    static CamColChk sCamColChk;
```
?

Reply:
yeah in how I name things I add underscores to the names I'm unsure of, the more underscores the less sure
I guess `sCamColChk` can difficultly be something else but `Camera_GetPitchAdjFromFloorHeightDiffs`?

Reply:
For `sCamColChk`, because there are many `CamColChk`'s, maybe something like `sPitchAdjColChk`? I'm also wonderin if the struct `CamColChk` is too abbreviated to begin with. Maybe just `CameraCollision` or something? Outside the scope of this PR though